apps/src/music/views/Controls.jsx

```diff
@@ -59,56 +60,83 @@ const Controls = ({
     );
   };
 
-  const renderIconButton = (icon, onClick) => (
-    <div className={classNames(moduleStyles.controlButtons, moduleStyles.side)}>
-      <FontAwesome
-        icon={icon}
-        className={moduleStyles.iconButton}
-        onClick={onClick}
-      />
-    </div>
+  const renderIconButton = (icon, onClick, hide) => (
+    <button
+      className={classNames(
+        moduleStyles.controlButton,
```

Comment:
The button's default `font-size` of `large` (which maps to `18px`) feels a bit jumbo compared to everything else on the page right now.  I'd be inclined to use `14px` for now.  And then also reduce the icon to suit that size.